### PR TITLE
Add kubecon demo to xgboost_ames_housing directory

### PR DESCRIPTION
**Pull request by abcdefgs0324**
- add Kubecon EU to examples

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/examples/589)

### CODE REVIEW

File: xgboost_ames_housing/demo/fairing_util.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+from kubernetes import client
```
Comment (zhenghuiwang) [inline xgboost_ames_housing/demo/fairing_util.py]: Can this be checked into fairing and then imported in the notebooks here?

File: xgboost_ames_housing/demo/mockup-data-xgboost-build-train-deploy.ipynb [hunk 2]
```diff
@@ -0,0 +1,1015 @@
+{
```
Comment (zhenghuiwang) [inline xgboost_ames_housing/demo/mockup-data-xgboost-build-train-deploy.ipynb]: What is the difference between this file and `ames-xgboost-build-train-deploy.ipynb`?

File: xgboost_ames_housing/demo/deployment/model.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
+apiVersion: extensions/v1beta1
```
Comment (zhenghuiwang) [inline xgboost_ames_housing/demo/deployment/model.yaml]: Where is this yaml file used?

File: xgboost_ames_housing/demo/pipelines-notebook.ipynb [hunk 4]
```diff
@@ -0,0 +1,422 @@
+{
```
Comment (zhenghuiwang) [inline xgboost_ames_housing/demo/pipelines-notebook.ipynb]: Is this notebook contained in `ames-xgboost-build-train-deploy.ipynb` ? Do you need to check it in?

File: xgboost_ames_housing/demo/xgboost-high-level-apis.ipynb [hunk 5]
```diff
@@ -0,0 +1,1002 @@
+{
```
Comment (zhenghuiwang) [inline xgboost_ames_housing/demo/xgboost-high-level-apis.ipynb]: Is this contained in `ames-xgboost-build-train-deploy.ipynb` ?

File: xgboost_ames_housing/demo/Dockerfile [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+RUN cd /tmp && \
+    curl -LO  https://github.com/github/hub/releases/download/v2.11.2/hub-linux-amd64-2.11.2.tgz && \
+    tar -xvf hub-linux-amd64-2.11.2.tgz && \
+    mv hub-linux-amd64-2.11.2 /usr/local && \
+    ln -sf /usr/local/hub-linux-amd64-2.11.2/bin/hub /usr/local/bin/hub
+
+# Bake a copy of update_model.py into the image
+# Used for testing the code with one of jobs.
+COPY update_model.py .
+RUN mkdir -p /src/jlewi/kubecon-demo && \
```
[the remark this answers is not in the document]
Comment (jlewi) [reply xgboost_ames_housing/demo/Dockerfile]: +1

**Comment by abcdefgs0324** [comment]
Several updates
- move example to its own directory
- leave only one notebook using mockup synthetic data
- change the names related to the conference
- use kubeflow/fairing

File: xgboost_synthetic/fairing_util.py [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import json
```
Comment (zhenghuiwang) [inline xgboost_synthetic/fairing_util.py]: Is this file intended to be checked in fairing? If so, can you merge it into fairing? If not, can you move it to `util.py`?

Comment (abcdefgs0324) [reply xgboost_synthetic/fairing_util.py]: I have deleted functions which are checked in fairing and the remaining one wound not. I will move it to `util.py`. Thanks!

**Comment by jlewi** [comment]
@abcdefgs0324 There are a bunch of lint failures in xgboost_util.py That's why test is failing;
look at the link
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/kubeflow_examples/589/kubeflow-examples-presubmit/1150994674568138752/

**Comment by abcdefgs0324** [comment]
@jlewi Fixed it.
